lms/djangoapps/bulk_enroll/serializers.py

```diff
@@ -43,3 +45,19 @@ def validate_courses(self, value):
             except InvalidKeyError:
                 raise serializers.ValidationError("Course key not valid: {}".format(course))
         return value
+
+    def validate(self, data):
+        """
+        Check that the cohorts list is the same size as the courses list.
+        """
+        if 'cohorts' in data and len(data['cohorts']) > 0:
```

Comment:
It is better to be strict with input. I would recommend removing the `len(data['cohorts']) > 0` clause.

Reply:
Since cohorts is an optional parameter, that will cause all requests that don't contain the cohorts parameter to fail.

Reply:
I meant just making this `if 'cohorts' in data:`. Clients should not be sending in this field if it is going to be empty.

Reply:
ok. Removed. Will push in a minute.